pages/[path].js

```diff
@@ -12,38 +12,45 @@ import QuotesWrapper from "@/components/QuotesWrapper/QuotesWrapper";
 import SmileTrainerWrapper from "@/components/SmileTrainerWrapper/SmileTrainerWrapper";
 import GuidedMeditation from "@/components/GuidedMeditation/GuidedMeditation";
 import Entry from "@/components/SingleEntry/Entry";
+import { useDashboardState } from "@/components/DashboardStateProvider/DashboardStateProvider";
+import { useSphereState } from "@/components/ContextProviders/SphereStateProvider/SphereStateProvider";
 
 export default function Home() {
+  const { dashboardIsOpen, handleDashboardIsOpen } = useDashboardState();
+  const { sphereState, handleSphereState } = useSphereState();
```

Comment:
```suggestion
  const { sphereState } = useSphereState();
```
Maybe I overlooked it, but it seems like you don't use the handleSphereState here?

Reply:
I give it to <ActionBar>